lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -169,7 +170,7 @@
   end
 
   execute("git-remove-delivery-config-branch") do
-    command("git branch -d add-delivery-configuration")
+    command("git branch -D add-delivery-configuration")
```

Comment:
When it's targeting a branch other than master, it fails because it's not merged into master.